spec/models/article_spec.rb

```diff
@@ -1,18 +1,15 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
 RSpec.describe Article, type: :model do
+  # `include` done here in order to avoid
+  # namespace collision with RegistrationController
+  include FactoryBot::Syntax::Methods
   describe 'creation' do
```

Comment:
can you please use let instead of variables, it is practice for Rspec

https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let

```
# frozen_string_literal: true

require 'rails_helper'

RSpec.describe Article, type: :model do
  include FactoryBot::Syntax::Methods

  describe 'creation' do
    let(:user) { create(:user) }
    let(article) { create(:article, user: user) }

    it 'can be created' do
      expect(article).to be_valid
    end
  end
end
```

Reply:
Done, all tests for both controllers and models are now using `let`.